app/src/userpages/components/CanvasPage/List/index.jsx

```diff
@@ -270,8 +274,18 @@ class CanvasList extends Component<Props, State> {
                                     }}
                                 >
                                     <Tile.Title>{canvas.name}</Tile.Title>
-                                    <Tile.Description>{new Date(canvas.updated).toLocaleString()}</Tile.Description>
-                                    <Tile.Status>{capital(canvas.state)}</Tile.Status>
+                                    <Tile.Description>
+                                        <Translate value="userpages.canvases.updated" since={moment(canvas.updated).fromNow()} />
+                                    </Tile.Description>
+                                    <Tile.Status
+                                        className={
+                                            cx({
+                                                [styles.running]: canvas.state === 'RUNNING',
+                                                [styles.stopped]: canvas.state === 'STOPPED',
```

Comment:
`RUNNING` and `STOPPED` can be taken from [`RunStates` in `$editor/canvas/state.js`](https://github.com/streamr-dev/streamr-platform/blob/PLATFORM-509-style-fixes/app/src/editor/canvas/state.js#L51-L54).